app/components/account/details/account-quotas-card/account-quotas-card.component.ts

```diff
@@ -1,37 +1,247 @@
-import { Component, Input, OnChanges } from "@angular/core";
-
-import { AccountResource } from "app/models";
-import { ComputeService } from "app/services";
+import { Component, Input, OnChanges, OnDestroy } from "@angular/core";
+import { Subscription } from "rxjs";
 
+import { LoadingStatus } from "app/components/base/loading";
+import { AccountResource, Pool } from "app/models";
+import { ComputeService, PoolListParams, PoolService, VmSizeService } from "app/services";
+import { ListView } from "app/services/core";
 import { ComponentUtils } from "app/utils";
+
+import { List } from "immutable";
+import { Observable } from "rxjs/Observable";
 import "./account-quotas-card.scss";
 
+type ProgressColorClass = "high-usage" | "medium-usage" | "low-usage";
+const additionalVmSizeCores = {
```

Comment:
Can you move that in either vm-size.service or constants and call it cloudServiceCores